rpc/client.go

```diff
@@ -30,17 +30,17 @@ func NewClient(addr string) (*Client, error) {
 
 // AddOrders adds orders to the 0x Mesh node and broadcasts them throughout the
 // 0x Mesh network.
-func (c *Client) AddOrders(orders []*zeroex.SignedOrder) (*AddOrdersResponse, error) {
-	var addOrdersResponseJSON string
-	if err := c.rpcClient.Call(&addOrdersResponseJSON, "mesh_addOrders", orders); err != nil {
+func (c *Client) AddOrders(orders []*zeroex.SignedOrder) (*zeroex.ValidationResults, error) {
+	var validationResultsJSON string
+	if err := c.rpcClient.Call(&validationResultsJSON, "mesh_addOrders", orders); err != nil {
```

Comment:
The `go-ethereum/rpc` package already does automatic marshaling/unmarshaling. I think you can just write it this way?

```go
var validationResults zeroex.ValidationResults
if err := c.rpcClient.Call(&validationResults, "mesh_addOrders", orders); err != nil {
    // ...
}
```